install-dependencies.sh

```diff
@@ -259,6 +259,37 @@ minio_download_jobs() {
     rm -f ${cfile}
 }
 
+open_search_arch() {
+    local -A OPEN_SEARCH_ARCH=(
+        ["x86_64"]=x64
+        ["aarch64"]=arm64
+    )
+    echo ${OPEN_SEARCH_ARCH["$(arch)"]}
+}
+
+OPEN_SEARCH_VERSION=2.18.0
+declare -A OPEN_SEARCH_CHECKSUM=(
+    ["x86_64"]=ae3cb4107b2e0cdbb9b98bb4e5f2f019b736b68e995442c718e459d39ff01df1
```

Comment:
Can the checksum be curled instead of hardcoded?

Referring to minio again, `install-dependencies.sh` is doing:
`echo $(curl -s "$(minio_server_url).sha256sum" | cut -f1 -d' ') "${MINIO_BINARIES_DIR}/minio" > $cfile`